src/coreclr/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -1583,8 +1583,7 @@ internal static void ValidateObject(object obj, IntPtr pMD)
         internal static partial void ValidateByref(IntPtr byref, IntPtr pMD); // the byref is pinned so we can safely "cast" it to IntPtr
 
         [Intrinsic]
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern IntPtr GetStubContext();
+        internal static IntPtr GetStubContext() => throw new UnreachableException(); // Unconditionally expanded intrinsic
```

Comment:
I have switched a few intrinsic-only FCalls to managed only impl while I was on it.